e2e/test/scenarios/question/notebook-data-source.cy.spec.ts

```diff
@@ -74,7 +74,7 @@ describe("scenarios > notebook > data source", () => {
       },
     );
 
-    it.skip("should display tables from the only existing database by default on an enterprise instance without token activation (metabase#40223)", () => {
+    it("should display tables from the only existing database by default on an enterprise instance without token activation (metabase#40223)", () => {
       onlyOnEE();
```

Comment:
@kamilmielnik @nemanjaglumac do you know any reason why do we want to test it only on EE?

Reply:
No idea. Let's just [close the issue](https://github.com/metabase/metabase/issues/40223#issuecomment-2189151258)?

Reply:
Because the original issue was reproducible only on an enterprise instance without an activated license.
But as we concluded [here](https://github.com/metabase/metabase/issues/40223#issuecomment-2189044871), the issue should be closed and the repro deleted.

It's no longer valid due to "Examples" we ship. Every instance will now always have saved question to begin with.

Reply:
updated PR